lighthouse-core/test/lib/page-functions-test.js

```diff
@@ -12,6 +12,92 @@ const pageFunctions = require('../../lib/page-functions.js');
 
 /* eslint-env jest */
 
+describe('createEvalCode', () => {
+  it('iife basic', () => {
+    function mainFn() {
+      return true;
+    }
+    const code = pageFunctions.createEvalCode(mainFn, {
+      mode: 'iife',
+    });
+    expect(code).toMatchInlineSnapshot(`
```

Comment:
to channel my inner @patrickhulce, since these are static, should just use `expect(code).toEqual(...)`

Reply:
I will say on the spectrum of obvious `toMatchInlineSnapshot` to obvious `toEqual`, this particular test does feel somewhat ambiguous to me (specific whitespace seems likely to change with no correct answer, but other important bits never should for example), but I love the reinforcement of the rule so much I'll go along with it 😆